src/sgame/sg_main.cpp

```diff
@@ -2273,6 +2276,10 @@ can see the last frag.
 */
 void CheckExitRules()
 {
```

Comment:
In particular this comment: https://github.com/DaemonEngine/Daemon/blob/master/src/shared/CommonProxies.cpp#L370